src/random_test.cpp

```diff
@@ -202,6 +202,10 @@ int sum_of_all_options(Thd1 *thd) {
     opt_int_set(ALTER_DATABASE_ENCRYPTION, 0);
   }
 
+  /* If OS is Mac, disable table compression as hole punching is not supported on OSX */
+  if (strcmp(PLATFORM_ID, "Darwin") == 0)
```

Comment:
Can we add some messages in log files as well. 
Table compression is disabled as hole punching is not supported on OSX

Reply:
Sure, we can add it. But, it will good if we do the same for all previous disabled features as well.
for ex.
we disabled a lot of features when Storage engine is RocksDB
we disabled column compression when Build is upstream version

Reply:
we can fix them in separate ticket. make a low priority ticketr. 
 for now adding some message is sufficent.

Reply:
Fixed